jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1260,11 +1281,20 @@ ecma_builtin_json_serialize_property (ecma_json_stringify_context_t *context_p,
   /* 11. */
   if (ecma_is_value_object (value) && !ecma_op_is_callable (value))
   {
+    ecma_value_t is_array = ecma_is_value_array (value);
+
+#if ENABLED (JERRY_ES2015)
+    if (ECMA_IS_VALUE_ERROR (is_array))
+    {
+      return is_array;
```

Comment:
The object in 'value' is leaking here. Please add a test case where the IsArray operation throws.